web/client/components/mapcontrols/annotations/Annotations.jsx

```diff
@@ -208,12 +208,13 @@ class Annotations extends React.Component {
 
     renderFieldValue = (field, annotation) => {
         const fieldValue = annotation.properties[field.name] || '';
-        switch (field.type) {
-        case 'html':
-            return <span dangerouslySetInnerHTML={{__html: fieldValue} }/>;
-        default:
-            return fieldValue;
+        if (field.type === 'html') {
+            // Return the text content of the first child of the html string (to prevent collating all texts into a single word)
+            return (new DOMParser).parseFromString(fieldValue, "text/html").documentElement.lastElementChild
+                ?.firstChild
+                ?.textContent || '';
```

Comment:
have you tried it in different browsers? 
if not can you do at least chrome, firefox, and edge?

Reply:
Yes @MV88 , I did check this on Chrome, Firefox and Edge. 

> (new DOMParser).parseFromString

**Supports** 
Chrome v31 and above
FireFox v12 and above
Edge v12 and above
IE v10 and above